ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/parameters/EnginePayloadParameter.java

```diff
@@ -64,7 +65,8 @@ public EnginePayloadParameter(
       @JsonProperty("logsBloom") final LogsBloomFilter logsBloom,
       @JsonProperty("prevRandao") final String prevRandao,
       @JsonProperty("transactions") final List<String> transactions,
-      @JsonProperty("withdrawals") final List<WithdrawalParameter> withdrawals) {
+      @JsonProperty("withdrawals") final List<WithdrawalParameter> withdrawals,
+      @JsonProperty("deposits") final List<DepositParameter> deposits) {
```

Comment:
The `DepositParameter` was created in the previous PR comparing the json structure against the Engine API spec for 6110 and but I think the field names are different.

It has `pubKey` instead of `pubkey` from spec and `withdrawal_credentials` instead of `withdrawalCredentials` from spec.